test/fixtures/for/nested.js

```diff
@@ -1,9 +1,9 @@
 let x
-for i from 0 til 10:
-  for j from 0 til 10:
+for let i=0; i<10; i++:
```

Comment:
though, now that I write that, I realize we should really add the caching in the output:

```
for (let i = 0, _arr = Array(10), _l = _arr.length; i < l; i++) 
```

Though that really doesn't need to be done immediately (and `_arr` should only be generated when it's either a `CallExpression` or a `MemberExpression` which contains a `CallExpression`, as with `for-in`, which it sounds like you're working on soon).